python/pip-generic-gentoo2023.conf

```diff
@@ -0,0 +1,17 @@
+[global]
+disable-pip-version-check = true
+
+[download]
+find-links = /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo2023/sse3 /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/sse3 /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo2023/generic /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic
+
+[wheel]
+find-links = /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo2023/sse3 /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/sse3 /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo2023/generic /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic
+disable-pip-version-check = true
+
+[install]
+find-links = /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo2023/sse3 /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/sse3 /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo2023/generic /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic /cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic
+constraint = /cvmfs/soft.computecanada.ca/config/python/constraints.txt
```

Comment:
This is for python 2, will we have a python 2 module in 2023? 
It does not hurt to have it there, but otherwise we could remove it.